tests/commands/test_rename.py

```diff
@@ -20,3 +19,37 @@ def test_rename_run(tmp_library: TemporaryLibrary) -> None:
     doc, = db.query_dict({"title": old_title})
     assert doc.get_main_folder_name() == new_name
     assert os.path.exists(doc.get_main_folder())
+
+
+def test_regenerate_name(tmp_library: TemporaryLibrary,
+                         tmp_config: TemporaryConfiguration,
+                         ) -> None:
+    from click.testing import CliRunner
```

Comment:
There should be a `papis.testing.PapisRunner` that wraps `CliRunner` and sets `mix_stderr=False`, so it's better to use that.

Reply:
TIL! Thanks, I'll use that!